libraries/Microsoft.Bot.Connector/Teams/TeamsOperations.cs

```diff
@@ -268,28 +236,22 @@ public async Task<HttpOperationResponse<TeamsMeetingParticipant>> FetchParticipa
                 throw new ValidationException(ValidationRules.CannotBeNull, nameof(tenantId));
             }
 
-            // Tracing
-            bool shouldTrace = ServiceClientTracing.IsEnabled;
-            string invocationId = null;
-            if (shouldTrace)
+            var content = new
             {
-                invocationId = ServiceClientTracing.NextInvocationId.ToString(CultureInfo.InvariantCulture);
-                Dictionary<string, object> tracingParameters = new Dictionary<string, object>();
-                tracingParameters.Add("meetingId", meetingId);
-                tracingParameters.Add("participantId", participantId);
-                tracingParameters.Add("tenantId", tenantId);
-                tracingParameters.Add("cancellationToken", cancellationToken);
-                ServiceClientTracing.Enter(invocationId, this, "GetParticipant", tracingParameters);
-            }
+                meetingId,
+                participantId,
+                tenantId,
+            };
+
+            var invocationId = TraceActivity("GetParticipant", content, cancellationToken);
```

Comment:
It shouldnt be FetchParticipants?

Reply:
I'm keeping the original name, I didn't create this method, maybe the name is for one reason